source/index.html.md.erb

```diff
@@ -31,6 +31,12 @@ Products at GDS in discovery or alpha development phases must [follow instructio
 
 Products in beta and live phases must follow both the [instructions set out in the Service Manual](https://www.gov.uk/service-manual/agile-delivery) and the standards in this repository.
 
+## Ways of working
+
+GDS works on the highest priorities while supporting its live services and adjusting its plans when priorities change. To do this GDS uses scaled Agile to plan and carry out its work in a quarterly planning cycle, find out how:
```

Comment:
can you change "scaled Agile" to "Agile at scale" to avoid any potential confusion